presto-product-tests/src/modernizer/exclusions.txt

```diff
@@ -0,0 +1 @@
+com/google/inject/Inject
```

Comment:
No, because `javax.inject.Inject` (JSR-330) doesn't support optional injection. I don't see any simple alternative -- getting rid of this seems like a major change to how tests are constructed (with little or no benefit).